spec/parsers/bulkrax/bagit_parser_spec.rb

```diff
@@ -5,6 +5,34 @@
 
 module Bulkrax
   RSpec.describe BagitParser do
+    describe '#total' do
+      context 'while exporting' do
```

Comment:
since we can export with various restrictions (all, importer, collection...) I think this context would do well with some greater specificity. 🙂